policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/basic_personal_amount.yaml

```diff
@@ -0,0 +1,13 @@
+description: Quebec provides this Basic Personal Amount.
+values:
+  2022-01-01: 16_143
+metadata:
+  unit: currency-CAD
+  period: year
+  label: Quebec Basic Personal Amount
+  reference:
+    - title: Quebec 2022 Income Tax Return - Non-refundable tax credits - Line 350
+      href: https://www.revenuquebec.ca/documents/en/formulaires/tp/2022-12/TP-1.D-V%282022-12%29.pdf#page=3
+    - title: Revenu Quebec – Basic Personal Amount (Line 350)
+      href: https://www.revenuquebec.ca/en/citizens/income-tax-return/completing-your-income-tax-return/how-to-complete-your-income-tax-return/line-by-line-help/350-to-398-1-non-refundable-tax-credits/line-350/
```

Comment:
is there a legal code (quebec income tax act f.e.) that we can cite as well - just to have some legal code

Reply:
Correct me if I'm wrong. I guess the Basic Personal Amount (BPA) is the same as the Personal Tax Credit?

https://www.legisquebec.gouv.qc.ca/en/document/cs/I-3?langCont=fr#se:752_0_0_1